app/routes/signup.js

```diff
@@ -1,6 +1,10 @@
+import Ember from 'ember';
 import Route from 'ember-route';
 
 export default Route.extend({
+  session: Ember.inject.service(),
+  store: Ember.inject.service(),
+
```

Comment:
I think you can get rid of the Ember import and these service injections now that the component is handling login.